bsonjs/bson/bson-error.c

```diff
@@ -109,8 +109,6 @@ bson_strerror_r (int err_code,  /* IN */
    if (strerror_s (buf, buflen, err_code) != 0) {
       ret = buf;
    }
-#elif defined(__GNUC__) && defined(_GNU_SOURCE)
-   ret = strerror_r (err_code, buf, buflen);
 #else /* XSI strerror_r */
```

Comment:
As of glibc 2.13, we can assume XSI compliance.  manylinux_2014 includes gcc 2.17.